src/MessageManager.cpp

```diff
@@ -208,7 +208,7 @@ void MessageManager::Unsubscribe( IMessageSubscriber* pSubscriber, MessageID m )
 void MessageManager::Broadcast( Message &msg ) const
 {
 	// GAMESTATE is created before MESSAGEMAN, and has several BroadcastOnChangePtr members, so they all broadcast when they're initialized.
-	if(this != NULL && m_Logging)
+	if(m_Logging) // 'this' won't be NULL
```

Comment:
And until MESSAGEMAN is actually created, the global variable is NULL, which means that when stuff calls it during the creation of GAMESTATE, the function is called on a null pointer.  (which you could argue is a problem in itself, but of a different nature)